tests/visualization_tests/matplotlib_tests/test_optimization_history.py

```diff
@@ -139,12 +149,43 @@ def objective(trial: Trial) -> float:
             return 0.0
         return 0.0
 
+    def objective_for_bars(trial: Trial) -> float:
+        return trial.suggest_float("x", 0, 1)
+
     # Test with trials.
     studies = [create_study(direction=direction) for _ in range(n_studies)]
     for study in studies:
         study.optimize(objective, n_trials=3)
+
     figure = plot_optimization_history(studies, error_bar=True)
     assert len(figure.get_lines()) == 4
+    assert list(figure.get_lines()[-1].get_xdata()) == [0, 1, 2]
+    if direction == "minimize":
+        assert list(figure.get_lines()[-1].get_ydata()) == [1.0, 1.0, 0.0]
+    else:
+        assert list(figure.get_lines()[-1].get_ydata()) == [1.0, 2.0, 2.0]
+
+    # Test for error_bar.
+    studies = [create_study(direction=direction) for _ in range(3)]
+    for idx, study in enumerate(studies):
+        if idx == 0:
+            study.enqueue_trial({"x": 0.10})
+        elif idx == 1:
+            study.enqueue_trial({"x": 0.30})
+        elif idx == 2:
+            study.enqueue_trial({"x": 0.20})
+        study.optimize(objective_for_bars, n_trials=1)
+    figure = plot_optimization_history(studies, error_bar=True)
+
+    assert_almost_equal(
+        np.array(figure.get_lines()[0].get_ydata()), np.array([0.2000000000]), decimal=10
+    )
+    assert_almost_equal(
+        np.array(figure.get_lines()[1].get_ydata()), np.array([0.1183503419]), decimal=10
+    )
+    assert_almost_equal(
+        np.array(figure.get_lines()[2].get_ydata()), np.array([0.2816496580]), decimal=10
+    )
```

Comment:
How about this?
```suggestion
    suggested_params = [0.1, 0.3, 0.2]
    for idx, (x, study) in enumerate(zip(suggested_params, studies)):
        study.enqueue_trial({"x": x})
        study.optimize(objective_for_bars, n_trials=1)
    figure = plot_optimization_history(studies, error_bar=True)

    mean = np.mean(suggested_params)
    std = np.std(suggested_params)

    assert_almost_equal(figure.get_lines()[0].get_ydata(), mean)
    assert_almost_equal(figure.get_lines()[1].get_ydata(), mean - std)
    assert_almost_equal(figure.get_lines()[2].get_ydata(), mean + std)
```

Reply:
That sounds great! Thanks your nice comment.